chainer/initializers/__init__.py

```diff
@@ -74,9 +73,8 @@ def _get_initializer(initializer):
 
     if initializer is None:
         return LeCunNormal()
-    if numpy.isscalar(initializer):
-        return Constant(initializer)
-    if isinstance(initializer, numpy.ndarray):
+    if (isinstance(initializer, chainer.get_array_types()) or
+            numpy.isscalar(initializer)):
```

Comment:
A nitpick but please place `or` after the new line, as per PEP-8.

Reply:
Do you know if there is any setting for flake8 to pick this ones?  😆

Reply:
I haven't heard about any.
Actually it's not a strict rule, but suggestion:

https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
>In Python code, it is permissible to break before or after a binary operator, as long as the convention is consistent locally. For new code Knuth's style is suggested.

Reply:
Yeah, I am aware of it and try to apply it always, but sometimes I miss and would be great if flake could complain 😂